pkgs/by-name/ap/apriltag/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  opencv4,
+  python3,
+}:
+
+let
+  opencv4WithGtk = opencv4.override {
```

```suggestion
  opencv4WithGtk = python3Packages.opencv4.override {
```
`opencv4` isn't a top-level package.